ProcessMaker/Http/Controllers/Api/ExportController.php

```diff
@@ -19,16 +18,35 @@
 use ProcessMaker\Models\ProcessTemplates;
 use ProcessMaker\Models\Screen;
 use ProcessMaker\Models\Script;
+use ProcessMaker\PackageHelper;
 
 class ExportController extends Controller
 {
+    const DATA_SOURCE_CLASS = 'ProcessMaker\Packages\Connectors\DataSources\Models\DataSource';
+
+    const DATA_SOURCE_EXPORTER_CLASS = 'ProcessMaker\Packages\Connectors\DataSources\ImportExport\DataSourceExporter';
+
+    const DECISION_TABLE_CLASS = 'ProcessMaker\Package\PackageDecisionEngine\Models\DecisionTable';
+
+    const DECISION_TABLE_EXPORTER_CLASS = 'ProcessMaker\Package\PackageDecisionEngine\ImportExport\DecisionTableExporter';
+
     protected array $types = [
         'screen' => [Screen::class, ScreenExporter::class],
         'process' => [Process::class, ProcessExporter::class],
         'script' => [Script::class, ScriptExporter::class],
         'process_templates' => [ProcessTemplates::class, TemplateExporter::class],
     ];
 
+    public function __construct()
+    {
+        if (PackageHelper::isPackageInstalled(self::DATA_SOURCE_CLASS)) {
+            $this->types['datasource'] = [self::DATA_SOURCE_CLASS, self::DATA_SOURCE_EXPORTER_CLASS];
+        }
+        if (PackageHelper::isPackageInstalled(self::DECISION_TABLE_CLASS)) {
+            $this->types['decisiontable'] = [self::DECISION_TABLE_CLASS, self::DECISION_TABLE_EXPORTER_CLASS];
```

Comment:
This PR has already been approved, but I wonder if not following the snake_case convention could bring problems in the future? 

- `datasource` - `data_source` and `decisiontable` - `decision_table`.

Also, keep in mind that from the front-end it is sent as kebab-case:

`<add-to-project-modal id="add-to-project-modal" ref="add-to-project-modal"  assetType="data-source" :assetId="assetId" :assetName="assetName" />`


cc: @sanjacornelius @estebangallego

Reply:
Resolved! Thank you @eiresendez